models/database.go

```diff
@@ -549,6 +549,10 @@ var databaseSchema = [][]string{
 		`ALTER TABLE agents
 		ADD COLUMN agent_password VARCHAR CHECK (agent_password <> '')`,
 	},
+	43: {
+		`ALTER TABLE artifacts
+      ADD COLUMN mode VARCHAR NOT NULL CHECK (mode <> '') DEFAULT 'snapshot'`,
```

Comment:
below it's said that we should avoid default values, so let's drop it after?

Reply:
Yep, added.